packages/@ama-sdk/schematics/package.json

```diff
@@ -40,6 +40,7 @@
     "@angular-devkit/core": "^16.0.5",
     "@angular-devkit/schematics": "^16.0.5",
     "@angular-devkit/schematics-cli": "^16.0.5",
+    "@o3r/schematics": "workspace:^",
```

Comment:
Then is should be provided by `@ama-sdk/create`

Reply:
added